### DISCUSSION TITLE
Org-copy-visible integration with evil-yank?

### DISCUSSION THREAD

**Discussion by rieje** [emacs-evil/evil-collection#731] [Ideas]
[EDIT: someone shared a [hack](https://0x0.st/HZ6o.txt) that replaces evil-yank functions which seems to work well except in both `yy` (yank to unnamed register) and `"y` (yank to system clipboard, it will `org-copy-visible` text. I only want to `org-copy-visible` for `"y` (system clipboard), since when I'm working with org-file and copying and pasting within it, I still want to keep the org-emphasized text.

--------

Anyone know how to integrate org-copy-visible with evil-yank? This is useful for copying org-emphasized (styled) text and e.g. pasting to terminal or document. 99% of the time you don't want to copy the emphasis markers that surround the text and only want to copy the text just as you see it. 

Basically, in org-mode, where this is relevant, overload the evil-yank function to org-copy-visible into the clipboard with the behavior of evil-yank (e.g. the option to copy to system clipboard if you want). As is, org-copy-visible does not copy to system clipboard.
